service/controller/release/resource_set.go

```diff
@@ -49,6 +63,7 @@ func NewResourceSet(config ResourceSetConfig) ([]resource.Interface, error) {
 	resources := []resource.Interface{
 		statusResource,
 		appsResource,
+		configsResource,
```

Comment:
Just to do this now already: I think this resource should run before the apps resource.